src/main/java/it/gov/pagopa/pu/pagopapayments/service/synchronouspayments/SynchronousPaymentService.java

```diff
@@ -86,8 +93,22 @@ public long retrieveNotificationFeeCents(Long organizationId, String nav, String
         return 0;
       }
     } else {
-      //TODO - P4ADEV-2694 if SENDApiKey doesn't exists call external third part API to retrieve notificationFee
-      return 0;
+      try{
```

Comment:
we should change actualization order:
if `dpTypeOrg.flagAmountActualization` is true, we should actualize using SIL
otherwise, if there is a SEND apiKey on the organization we should actualize using  SEND.

@robertopasetti , @serdimic what should we do if the `dpTypeOrg.flagAmountActualization` is null, but there is not an `amountActualizationOrgSilServiceId`?